lib/flight.js

```diff
@@ -124,3 +124,13 @@ Flight.prototype.rightFlip = function() {
 Flight.prototype.wave = function() {
   return this.connection.animate("wave", 750);
 };
+
+/**
+ * Tells the drone to perform horizontal trimming.
+ *
+ * @return {null}
+ * @publish
+ */
+Flight.prototype.ftrim = function() {
```

Comment:
You are correct, just adding it to `commands.js` will cause `ftrim` to be proxied to the underlying ardrone connection https://github.com/hybridgroup/cylon-ardrone/blob/master/lib/ardrone.js#L114